tests/Oryx.Integration.Tests/Python/PythonCustomStartUpCommandTest.cs

```diff
@@ -0,0 +1,124 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Oryx.BuildScriptGenerator.Python;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "python")]
+    public class PythonCustomStartUpCommandTest : PythonEndToEndTestsBase
+    {
+        public PythonCustomStartUpCommandTest(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
+            : base(output, testTempDirTestFixture)
+        {
+        }
+
+        [Theory]
+        [InlineData("2.7")]
+        [InlineData("3.6")]
+        [InlineData("3.7")]
+        public async Task CanBuildAndRunPythonApp_UsingCustomStartUpScript(string pythonVersion)
+        {
+            // Arrange
+            var appName = "http-server-py";
+            var volume = CreateAppVolume(appName);
+            var appDir = volume.ContainerDir;
+
+            var buildScript = new ShellScriptBuilder()
+                .AddCommand($"oryx build {appDir} --platform {PythonConstants.PlatformName} --platform-version {pythonVersion}")
+                .ToString();
+
+            // Create a custom startup script
+            const string customStartUpScript = "customStartup.sh";
+            File.WriteAllText(Path.Join(volume.MountedHostDir, customStartUpScript),
```

Comment:
may be its better to add this script as part of the sample app and adjust the filepath here.